### ISSUE TITLE
Deprecate on Atmosphere too

### ISSUE THREAD

**Issue by dandv** [CollectionFS/Meteor-http-publish#14]
`meteor admin set-unmigrated cfs/http-publish`

**Comment by raix** [comment]
@tmeasday I think we might need an official way of deprecating packages?
Eg.: `meteor admin deprecate cfs:http-publish` *Even if it's just an alias for set-unmigrate...*

**Comment by tmeasday** [comment]
Yeah, although `set-unmigrated` is sort of the defacto "delete package" command. I'm not sure it quite means deprecated though..?
